src/requestforattestation/RequestForAttestation.ts

```diff
@@ -108,21 +150,40 @@ export default class RequestForAttestation implements IRequestForAttestation {
   public constructor(
     claim: IClaim,
     legitimations: AttestedClaim[],
-    identity: Identity,
+    claimOwner: NonceHash,
+    ctypeHash: NonceHash,
+    claimHashTree: object,
+    identity?: Identity,
```

Comment:
you can just pass in the identity address directly

Reply:
We need the Identity for its `signStr()` method used in `ResquestForAttestation.sign()`